types/picomatch/lib/picomatch.d.ts

```diff
@@ -241,7 +241,7 @@ declare namespace picomatch {
         options?: PicomatchOptions,
         returnOutput?: boolean,
         returnState?: boolean,
-    ): typeof compileRe;
+    ): RegExp;
```

Comment:
Looks good. We could also use:

```suggestion
    ): ReturnType<typeof compileRe>;
```

Reply:
This feels like it's relying on implementation details, but happy to change it if you think that's best!